src/Sitecore.Modules.WeBlog.WebForms/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Sitecore.Modules.WeBlog")]
```

Comment:
As with the MVC assembly info file, make sure you've got "WebForms" in the assembly title.